app/views/base/layout.scala

```diff
@@ -258,17 +265,21 @@ object layout:
     frag(
       doctype,
       htmlTag(
+        (ctx.data.inquiry.isEmpty && ctx.impersonatedBy.isEmpty && !ctx.blind)
+          .option(cls := ctx.pref.themeColorClass),
         topComment,
         head(
           charset,
           viewport,
           metaCsp(csp),
           metaThemeColor,
+          systemThemeScript,
           st.headTitle:
             val prodTitle = fullTitle | s"$title • $siteName"
             if netConfig.isProd then prodTitle
             else s"${ctx.me.so(_.username + " ")} $prodTitle"
           ,
+          cssTag("theme-all"),
```

Comment:
i'm just including them all here for now (they compress down to < 5k). we can change it to download only the selected theme's color css though.